aws-lambda-function/locals.tf

```diff
@@ -18,7 +18,7 @@ locals {
     DD_ENV                    = var.env
     DD_SERVICE                = var.service_name
     DD_SERVICE_MAPPING        = "lambda_sqs:${var.service_name}"
-    DD_TAGS                   = "service:${var.service_name},git.commit.sha:${var.git_commit_hash},git.repository_url:${var.git_repository_url}"
+    DD_TAGS                   = "service:${var.service_name},git.repository_url:${var.git_repository_url}"
```

Comment:
Tohle vynucuje deployment lambda funkce při každým commitu, i když se nic v jejím kódu nezmění. Nevím jak to obejít, tak prozatím odstraním.